backend/spec/factories/agents.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :agent do
+    name { Faker::Name.name }
+    address { Faker::Address.full_address }
+    working_time { Faker::Lorem.sentence }
+    telephone_number { Faker::PhoneNumber.phone_number_with_country_code }
+    email { Faker::Internet.email }
+    photo_url { Faker::Internet.url }
+    slogan { Faker::Lorem.sentence }
+    access { Faker::Types.rb_array(len: rand(1..10)) }
```

Comment:
```suggestion
    access { Faker::Types.rb_array(len = rand(1..10)) }
```
https://www.rubydoc.info/github/stympy/faker/master/Faker%2FTypes.rb_array

ここ間違えたかも

Reply:
コンソルで実行してみたところ、このメッセージが出てきました。
```
Passing `len` with the 1st argument of `rb_array` is deprecated. Use keyword argument like `rb_array(len: ...)` instead.
```
`len = rand(1..10)`は廃止予定だと思います